Source/ProgressionSystemRuntime/Private/Components/PSSpotComponent.cpp

```diff
@@ -28,6 +27,11 @@ void UPSSpotComponent::OnInitialized_Implementation()
 {
 	UPSWorldSubsystem& WorldSubsystem = UPSWorldSubsystem::Get();
 	WorldSubsystem.OnCurrentActiveSaveRowChanged.AddUniqueDynamic(this, &ThisClass::OnCurrentActiveSaveRowChanged);
+	WorldSubsystem.OnCurrentScoreChanged.AddUniqueDynamic(this, &ThisClass::OnCurrentScoreChanged);
+
+	// Listen to handle input for each game state
```

Comment:
`Listen to handle input for each game state` comment looks like copypaste. Please remove it, or update

Reply:
added Listen to handle input for each game stat